app/code/Magento/InventoryLowQuantityNotification/Setup/InstallData.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryLowQuantityNotification\Setup;
+
+use Magento\CatalogInventory\Model\Stock\Item as StockItem;
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Inventory\Model\ResourceModel\SourceItem;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryCatalog\Api\DefaultSourceProviderInterface;
+use Magento\InventoryLowQuantityNotification\Setup\Operation\CreateSourceConfigurationTable;
+
+/**
+ * Copy notify_stock_qty data from cataloginventory_stock_item to inventory_low_stock_notification_configuration.
+ */
+class InstallData implements InstallDataInterface
+{
+    /**
+     * @var DefaultSourceProviderInterface
+     */
+    private $defaultSourceProvider;
+
+    public function __construct(DefaultSourceProviderInterface $defaultSourceProvider)
+    {
+        $this->defaultSourceProvider = $defaultSourceProvider;
+    }
+
+    /**
+     * {@inheritdoc}
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function install(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
```

Comment:
Need to add `<sequence>` on `InventoryCatalog` (we suppose that source items for default source are already created)